sec_edgar_downloader/_Downloader.py

```diff
@@ -67,6 +67,7 @@ def get(
         before: Optional[Date] = None,
         include_amends: bool = False,
         download_details: bool = False,
+        skip_accession_numbers: Optional[set[str]] = None,
```

Comment:
Please use Set from typing library here :)

Reply:
IMHO we should not expose this as a parameter, but instead automatically use the existing file system to determine which items to skip. 

Thoughts?